src/libraries/Common/src/System/Security/Cryptography/RSASecurityTransforms.cs

```diff
@@ -38,19 +44,7 @@ internal RSASecurityTransforms(SafeSecKeyRefHandle publicKey, SafeSecKeyRefHandl
                 SetKey(SecKeyPair.PublicPrivatePair(publicKey, privateKey));
             }
 
-            public override KeySizes[] LegalKeySizes
-            {
-                get
-                {
-                    return new KeySizes[]
-                    {
-                        // All values are in bits.
-                        // 1024 was achieved via experimentation.
-                        // 1024 and 1024+8 both generated successfully, 1024-8 produced errSecParam.
-                        new KeySizes(minSize: 1024, maxSize: 16384, skipSize: 8),
-                    };
-                }
-            }
+            public override KeySizes[] LegalKeySizes => [s_legalKeySize];
```

Comment:
while this looks correct, we seem to use [CloneKeySizesArray](https://github.com/dotnet/runtime/blob/f6e46ff86b784db1106e2148d84e43432f96116c/src/libraries/Common/src/System/Security/Cryptography/KeySizeHelpers.cs#L12) elsewhere - this will make potential down-porting easier